docs/050-breaking-changes.rst

```diff
@@ -431,7 +432,7 @@ New version:
 
 ::
 
-    pragma solidity >=0.5.0 <0.7.0;
+    pragma solidity >=0.6.0 <0.7.0;
```

Comment:
This is the "upgrade from 0.4 to 0.5"-part - we should not require 0.6 in this example.

Reply:
I agree with you, that is something I wanted to point out.
This part was updated with things which were released after 0.5.0.

Way to do it would be to make example aligned to 0.5.0?

Reply:
Ah sorry - is that the one with the `push()`? Yes, it should be compilable with 0.5.0

Reply:
It complains about: 

`Error: Expected identifier but got '('
receive() payable external {}
            ^`

Reply:
`I'll swap receive() payable external {}` with
`I'll swap function() payable external {}` as it was originally, `receive` and `fallback` were introduced later

Reply:
ok!